src/Chapter1.hs

```diff
@@ -652,9 +665,9 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Integral a => a -> a
+firstDigit n = if (abs n) < 10 then (abs n)
+               else firstDigit ((abs n) `div` 10)
```

Comment:
That is right 💯 

You can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼

Reply:
The `where` case has become one of my favorites as I've continued doing these exercises. Thank you for taking the time to review, and thanks for putting this project out there!